docs/index-en.html

```diff
@@ -0,0 +1,4245 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+    <link rel="stylesheet" href="resources/primer.css" media="screen">
+    <link rel="stylesheet" href="resources/rec.css" media="screen">
+    <link rel="stylesheet" href="resources/extra.css" media="screen">
+    <link rel="stylesheet" href="resources/owl.css" media="screen">
+    <title>OntoUML Metamodel Vocabulary</title>
+    <!-- SCHEMA.ORG METADATA -->
+    <script type="application/ld+json">{"@context":"https://schema.org","@type":"TechArticle","url":"https://w3id.org/ontouml#","image":"http://vowl.visualdataweb.org/webvowl/#iri=https://w3id.org/ontouml#","name":"OntoUML Metamodel Vocabulary", "headline":"A reference implementation of the OntoUML Metamodel in OWL.", "datePublished":"2023-07-23", "version":"General review for compliance with the OntoUML Metamodel 1.1.0.", "license":"https://creativecommons.org/licenses/by-sa/4.0", "author":[{"@type":"Person","name":"Tiago Prince Sales"},{"@type":"Person","name":"Claudenir M. Fonseca"},{"@type":"Person","name":"Pedro Paulo Favato Barcelos"}], "contributor":[{"@type":"Person","name":"Mattia Fumagalli"}]}</script>
+    <script src="resources/jquery.js"></script>
+    <script src="resources/marked.min.js"></script>
+    <script> 
+function loadHash() {
+  jQuery(".markdown").each(function(el){jQuery(this).after(marked(jQuery(this).text())).remove()});
+	var hash = location.hash;
+	if($(hash).offset()!=null){
+	  $('html, body').animate({scrollTop: $(hash).offset().top}, 0);
+}
+	loadTOC();
+}
+function loadTOC(){
+	//process toc dynamically
+	  var t='<h2>Table of contents</h2><ul>';i = 1;j=0;
+	  jQuery(".list").each(function(){
+		if(jQuery(this).is('h2')){
+			if(j>0){
+				t+='</ul>';
+				j=0;
+			}
+			t+= '<li>'+i+'. <a href=#'+ jQuery(this).attr('id')+'>'+ jQuery(this).ignore("span").text()+'</a></li>';
+			i++;
+		}
+		if(jQuery(this).is('h3')){
+			if(j==0){
+				t+='<ul>';
+			}
+			j++;
+			t+= '<li>'+(i-1)+'.'+j+'. '+'<a href=#'+ jQuery(this).attr('id')+'>'+ jQuery(this).ignore("span").text()+'</a></li>';
+		}
+	  });
+	  t+='</ul>';
+	  $("#toc").html(t); 
+}
+$(function(){
+    loadHash();
+}); $.fn.ignore = function(sel){
+        return this.clone().find(sel||">*").remove().end();
+ }; 
+   </script>
+  </head>
+  <body>
+    <div class="container">
+      <div class="head">
+        <div style="float:right">language <a href="index-en.html"><b>en</b></a>
+        </div>
+        <h1>OntoUML Metamodel Vocabulary</h1>
+        <h2>Release 2023-07-23</h2>
```

Comment:
We should either use semantic versioning or date-based versioning.

Reply:
This information is generated after the value we attributed to "dct:issued" in our metadata. We are using only semantic versioning. Note that this is part of the W3C template (their Recommendations always provide the release date on top). IMO, we should keep as it is. But, @tgoprince and @claudenirmf, if you decide to remove, I won't oppose.

Reply:
I substituted the date by the version and added the modification date together with the other metadata